Add task solution

k-marchuk opened the pull request:
- [DEMO LINK](https://k-marchuk.github.io/layout_landing-page/)

ArturErmolenko commented on src/styles/blocks/categories.scss:
```diff
@@ -0,0 +1,116 @@
+    position: relative;
+    transition: transform $effectDuration;
+
+    &:hover {
+      transform: scale(1.05);
+    }
+    &--wide {
+      @include on-tablet {
+        grid-column: span 4;
+      }
+
+      @include on-desktop {
+        grid-column: span 8;
+      }
+    }
+    &--square {
+      height: 0;
+      padding-bottom: 100%;
```
```suggestion
    &:hover {
      transform: scale(1.05);
    }
    
    &--wide {
      @include on-tablet {
        grid-column: span 4;
      }

      @include on-desktop {
        grid-column: span 8;
      }
    }
    
    &--square {
      height: 0;
      padding-bottom: 100%;
```

ArturErmolenko commented on src/styles/blocks/menu.scss:
```diff
@@ -0,0 +1,48 @@
+  }
+
+  &__top {
+    position: sticky; /* to keep cross always visible */
```
you can remove comments

ArturErmolenko commented on src/styles/blocks/menu.scss:
```diff
@@ -0,0 +1,48 @@
+    display: block;
+    margin-bottom: 16px;
+    font-size: 16px;
+    line-height: 22.4px;
```
```suggestion
    line-height: 22px;
```

ArturErmolenko commented on src/styles/blocks/categories.scss:
```diff
@@ -0,0 +1,116 @@
+    }
+
+    &:active {
+      background-color: #111;
+      color: #fff;
```
use variables

ArturErmolenko commented on src/styles/blocks/categories.scss:
```diff
@@ -0,0 +1,116 @@
+  &__title {
+    margin: 0;
+    font-size: 24px;
+    line-height: 33.6px;
```
```suggestion
    line-height: 33px;
```
use integers

ArturErmolenko commented on src/styles/blocks/footer.scss:
```diff
@@ -0,0 +1,126 @@
+
+  &:hover {
+    background-color: $footer-text-color;
+    color: #fff;
```
use variable

ArturErmolenko reviewed (approved):
Wow, looks great 🔥 🐝 
Awesome job 👍 
I left a few comments, not critical, but nice to have.

k-marchuk commented:
> Wow, looks great 🔥 🐝 Awesome job 👍 I left a few comments, not critical, but nice to have.

Thank you for the review! 
I fixed comments and updated pull request.
